spec/controllers/user_controller_spec.rb

```diff
@@ -783,13 +783,13 @@ def do_signin(email, password)
   end
 
   it 'accepts only whitelisted parameters' do
-    post :signup, { :user_signup => { :email => 'silly@localhost',
-                                      :name => 'New Person',
-                                      :password => 'sillypassword',
-                                      :password_confirmation => 'sillypassword',
-                                      :admin_level => 'super' } }
-
-    expect(assigns(:user_signup).admin_level).to eq('none')
+    expect {
```

Comment:
Explicitly disabled for `./spec` https://github.com/mysociety/alaveteli/pull/3350/commits/d57a1158dda59004d5624b06048ecb174c60c430